src/plugins/charts/bar/inspector/BarGraphInspectorViewProvider.js

```diff
@@ -41,7 +41,7 @@ export default function BarGraphInspectorViewProvider(openmct) {
           _destroy = destroy;
         },
         priority: function () {
-          return openmct.priority.HIGH + 1;
+          return openmct.editor.isEditing() ? openmct.priority.HIGH + 1 : openmct.priority.DEFAULT;
```

Comment:
`HIGH + 1` is a code smell to me. Should something else just be a lower priority?

Reply:
Turns out... these weren't needed!